app/models/ncr/work_order.rb

```diff
@@ -8,6 +8,13 @@ def self.table_name_prefix
 
   EXPENSE_TYPES = %w(BA60 BA61 BA80)
   BUILDING_NUMBERS = YAML.load_file("#{Rails.root}/config/data/ncr/building_numbers.yml")
+  FY16 = Time.zone.parse('2015-10-01')
```

Comment:
"Yes" to all of these things. Can call the Concern `PurchaseCardUseCase` or `PurchaseCardMixin` or `PurchaseCardRequest` or something.